app/views/users/_org_admin.html.erb

```diff
@@ -79,7 +79,7 @@
 <div class="crayons-card grid gap-6 p-6 mb-6">
   <h2>Organization details</h2>
 
-  <%= form_for @organization, html: { class: "grid gap-6" } do |f| %>
+  <%= form_for @organization, url: organization_path(params[:locale], @organization), html: { class: "grid gap-6" } do |f| %>
```

Comment:
Will we need to add this to all helpers across the site? Also will it matter in non `GET` methods?

Reply:
I'm not sure yet 🤔 . I'm still trying to find a way to not have to do this once I get the other specs fixed.

I don't think it should matter in non `GET` methods. We may need it with non `GET` too because otherwise, the server may interpret that the response should be in the default locale without one being provided.